api-gateway/src/main/resources/static/scripts/vet-details/vet-details.template.html

```diff
@@ -99,128 +103,135 @@ <h3 class="form-label-rating text-center" style="margin-top: 5%">Educations</h3>
                             <th>Field Of Study</th>
                             <th>Start Year</th>
                             <th>End Year</th>
-                            <th>Delete</th>
                         </tr>
                         </thead>
                         <tbody>
-
-                        <tr ng-repeat="educations in $ctrl.educations track by educations.educationId">
-                                <td>{{ educations.degree }}</td>
-                                <td>{{ educations.schoolName }}</td>
-                                <td>{{ educations.fieldOfStudy }}</td>
-                                <td>{{ educations.startDate }}</td>
-                                <td>{{ educations.endDate }}</td>
-                                <td>
-                                    <a class="btn btn-danger educationButton" href="javascript:void(0)" ng-click="deleteVetEducation(educations.educationId)">Delete Education</a>
-                                    </td>
-                            </tr>
+                        <tr ng-repeat="educations in $ctrl.educations track by educations.educatonId">
+                            <td>{{ educations.degree }}</td>
+                            <td>{{ educations.schoolName }}</td>
+                            <td>{{ educations.fieldOfStudy }}</td>
+                            <td>{{ educations.startDate }}</td>
+                            <td>{{ educations.endDate }}</td>
+                        </tr>
```

Comment:
```suggestion
                        <tr ng-repeat="educations in $ctrl.educations track by educations.educatonId">
                            <td>{{ educations.degree }}</td>
                            <td>{{ educations.schoolName }}</td>
                            <td>{{ educations.fieldOfStudy }}</td>
                            <td>{{ educations.startDate }}</td>
                            <td>{{ educations.endDate }}</td>
                            <td>
                                    <a class="btn btn-danger educationButton" href="javascript:void(0)" ng-click="deleteVetEducation(educations.educationId)">Delete Education</a>
                                    </td>
                            </tr>
```
Did you accidentally delete (hah) the delete button?

Reply:
I sure did, sorry!! Fixing it right now